Mathlib/MeasureTheory/Group/GeometryOfNumbers.lean

```diff
@@ -83,4 +83,56 @@ theorem exists_ne_zero_mem_lattice_of_measure_mul_two_pow_lt_measure [NormedAddC
   refine' h_conv hw (h_symm _ hv) _ _ _ <;> norm_num
 #align measure_theory.exists_ne_zero_mem_lattice_of_measure_mul_two_pow_lt_measure MeasureTheory.exists_ne_zero_mem_lattice_of_measure_mul_two_pow_lt_measure
 
+/-- The **Minkowski Convex Body Theorem for compact domain**. If `s` is a convex compact symmetric
+domain of `E` whose volume is large enough compared to the covolume of a lattice `L` of `E`, then it
+contains a non-zero lattice point of `L`. Compared to
+`exists_ne_zero_mem_lattice_of_measure_mul_two_pow_lt_measure`, this version requires in addition
+that `s` is compact and `L` is discrete but provides a weaker inequality rather than a strict
+inequality. -/
+theorem exists_ne_zero_mem_lattice_of_measure_mul_two_pow_le_measure [NormedAddCommGroup E]
+    [NormedSpace ℝ E] [BorelSpace E] [FiniteDimensional ℝ E] [Nontrivial E] [IsAddHaarMeasure μ]
+    {L : AddSubgroup E} [Countable L] [DiscreteTopology L] (fund : IsAddFundamentalDomain L F μ)
+    (h_symm : ∀ x ∈ s, -x ∈ s) (h_conv : Convex ℝ s) (h_cpt : IsCompact s) (h_mes : μ s ≠ 0)
```

Comment:
you shouldn't need `h_mes` here: if the measure of `s` were zero, then by `h` the measure of `F` would be zero, and then by `fund` the measure of the whole space would be zero, a contradiction.

Reply:
Indeed, I have a PR #8287 that proves that the volume of a fundamental domain is nonzero. Using it, I can remove the assumption.